src/WorkoutsMain/WorkoutList/WorkoutList.css

```diff
@@ -0,0 +1,27 @@
+.WorkoutList .container {
+    display: grid;
+    row-gap: 10px;
+    margin: 0 20px;
+}
+
+
+.WorkoutList .WorkoutListTitle {
+    text-align: left;
```

Comment:
text-align left is default value.

I would remove from here and delete text-align: center from App.js